icepyx/core/APIformatting.py

```diff
@@ -161,6 +212,35 @@ def combine_params(*param_dicts):
         params.update(dictionary)
     return params
 
+def to_string(params):
+    """
+    Combine a parameter dictionary into a single url string
+
+    Parameters
+    ----------
+    params : dictionary
+
+    Returns
+    -------
+    url string of input dictionary (not encoded)
+
+    Examples
+    --------
+    >>> CMRparams = {'short_name': 'ATL06', 'version': '002', 'temporal': '2019-02-20T00:00:00Z,2019-02-28T23:59:59Z', 'bounding_box': '-55,68,-48,71'}
+    >>> reqparams = {'page_size': 10, 'page_num': 1}
+    >>> params = icepyx.core.APIformatting.combine_params(CMRparams, reqparams)
+    >>> icepyx.core.APIformatting.to_string(params)
+    '&short_name=ATL06&version=002&=temporal=2019-02-20T00:00:00Z,2019-02-28T23:59:59Z&bounding_box=-55,68,-48,71&page_size=10&page_num=1'
+    """
+    param_list = []
+    for k,v in params.items():
+        if isinstance(v,list):
+            for l in v:
+                param_list.append(k+'='+l)
+        else:
+            param_list.append(k+'='+str(v))
+    # return the parameter string
+    return "&".join(param_list)
```

Comment:
This function is currently untested. I can try to add a test for it later this week.

Reply:
I'll make a test from the example I put in the docstring :)